guides/common/modules/ref_global-parameters-for-registration.adoc

```diff
@@ -0,0 +1,20 @@
+[id="global-parameters-for-registration_{context}"]
+= Global Parameters for Registration
+
+You can configure the following global parameters by navigating to *Configure* > *Global Parameters*:
+
+* The `host_registration_remote_execution` parameter is used in the `remote_execution_ssh_keys` snippet.
+If it is set to `true`, the registration enables remote execution on the host.
+The default value is `true`.
+* The `host_registration_insights` parameter is used in the `insights` snippet.
+ifdef::satellite[]
+The default value is `true`.
+endif::[]
+ifndef::satellite[]
+The default value is `false`.
+endif::[]
+Set the parameter type to `boolean` to override the parameter value.
```

Comment:
Please reword. type is boolean and value would then be false probably.

Reply:
The value depends on the default, which is different for Satellite and non-Satellite.